### ISSUE TITLE
Propose api repo deletion

### ISSUE THREAD

**Issue by endepointe** [SecurityLogMiner/log-collection-frontend#27]
The api repo handles requests from the client but that responsibility can be handled directly by svelte itself. 

Passing the current api repo's responsibility onto the frontend would make it simpler for the user to use the collection service and remove the middleman which is the api repo.

**Comment by murchej** [comment]
I agree. Let's see what of the api repo we can integrate into the front-end before deletion.

**Comment by Jicxer** [comment]
> I agree. Let's see what of the api repo we can integrate into the front-end before deletion.

I am unsure what the front end implementation can achieve as of right now. This can be brought up for discussion later.
I would agree though that if we don't need the API, having the front-end only to handle requests by the client simplifies things

**Comment by Jicxer** [comment]
Unable to Archive as Alvin has the admin account
